osf_tests/test_node.py

```diff
@@ -3951,7 +3951,7 @@ def test_empty_is_public(self):
         qs = AbstractNode.objects.can_view()
 
         assert proj in qs
-        assert qs.count() == 1
+        assert qs.count() == 2  # proj and user's quickfiles node
```

Comment:
Minor: It might make more sense to use `Node` rather than `AbstractNode` in these tests, so we can avoid dealing with QuickFiles.